core/roboconf-messaging-rabbitmq/src/main/java/net/roboconf/messaging/rabbitmq/RabbitMqClientDm.java

```diff
@@ -288,12 +302,28 @@ public void listenToTheDm( ListenerCommand command )
 	}
 
 
+	@Override
+	public String getMessagingType() {
+		return MessagingConstants.FACTORY_RABBIT_MQ;
+	}
+
+
+	@Override
+	public Map<String, String> getConfiguration() {
```

Comment:
There is the exact same method in RabbitMqClientAgent.java
Is it possible to put it in RabbitMqClient (the common super class)?